yarn.lock

```diff
@@ -6690,11 +6738,11 @@ tar@^4:
     yallist "^3.0.2"
 
 terminate@^2.1.0:
-  version "2.1.0"
-  resolved "https://registry.yarnpkg.com/terminate/-/terminate-2.1.0.tgz#a87ee424be01a1d28f2f301045043a5bcd679a05"
-  integrity sha1-qH7kJL4BodKPLzAQRQQ6W81nmgU=
+  version "2.1.2"
```

Comment:
`terminate` moved from 2.1.0 -> [2.1.2](https://github.com/dwyl/terminate/releases/tag/v2.1.2) which includes the fix: https://github.com/dwyl/terminate/issues/35